enterprise/server/quota/quota_manager.go

```diff
@@ -322,11 +322,10 @@ func (qm *QuotaManager) createNamespace(env environment.Env, name string, config
 // default bucket. Returns nil if the namespace is not found or the default bucket
 // is not defined.
 func (qm *QuotaManager) findBucket(namespace string, key string) Bucket {
-	qm.mu.Lock()
+	qm.mu.RLock()
```

Comment:
is this better than a https://pkg.go.dev/sync#Map

Reply:
Used sync.Map instead